lib/virtfs/ext3/alloc_bitmap.rb

```diff
@@ -1,16 +1,16 @@
-module Ext3
+module VirtFS::Ext3
   class AllocBitmap
     def initialize(data)
-      raise "Ext3::AllocBitmap.initialize: Nil data" if data.nil?
+      raise "Ext::AllocBitmap.initialize: Nil data" if data.nil?
```

Comment:
I was thinking so that there errors would be more generic / applicable to both the Ext3 and Ext4 modules. Not a big deal, can change it back if that is preferred